### ISSUE TITLE
Meet YOLO26: the latest stable Ultralytics YOLO 🚀

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/ultralytics#24844]
YOLO26 is the current recommended Ultralytics model generation for real-time vision AI: faster, more accurate, production-ready, and available across the core YOLO task family.

This pinned issue is the launch hub for users who want the latest Ultralytics models, examples, docs, and upgrade path in one place.

## What shipped

- **YOLO26 detection** with end-to-end, NMS-free inference support for simpler deployment paths.
- **Seven vision tasks** in one framework: detection, instance segmentation, semantic segmentation, depth estimation, classification, pose estimation, and oriented bounding boxes.
- **Five model scales**: `n`, `s`, `m`, `l`, and `x`, from edge devices to high-accuracy workloads.
- **Automatic weight downloads** from official Ultralytics releases on first use.
- **Production workflows** for train, val, predict, export, track, and benchmark.

## Start here

- YOLO26 docs: https://docs.ultralytics.com/models/yolo26
- Roadmap: https://www.ultralytics.com/roadmap
- Main repository: https://github.com/ultralytics/ultralytics
- Python usage: https://docs.ultralytics.com/usage/python
- CLI usage: https://docs.ultralytics.com/usage/cli
- Configuration: https://docs.ultralytics.com/usage/cfg
- Train: https://docs.ultralytics.com/modes/train
- Predict: https://docs.ultralytics.com/modes/predict
- Track: https://docs.ultralytics.com/modes/track
- Export: https://docs.ultralytics.com/modes/export
- Ultralytics Platform: https://platform.ultralytics.com/ultralytics/yolo26

## Quickstart: CLI

```bash
pip install -U ultralytics

# Predict with the smallest YOLO26 detection model
yolo predict model=yolo26n.pt source=https://ultralytics.com/images/bus.jpg imgsz=640

# Train on a small detection dataset
yolo train data=coco8.yaml model=yolo26n.pt epochs=10 imgsz=640

# Validate a pretrained model
yolo val data=coco8.yaml model=yolo26n.pt imgsz=640

# Export for deployment
yolo export model=yolo26n.pt format=onnx imgsz=640
```

## Quickstart: Python

```python
from ultralytics import YOLO

# Load an official YOLO26 model. Weights download automatically on first use.
model = YOLO("yolo26n.pt")

# Run inference
results = model("https://ultralytics.com/images/bus.jpg", imgsz=640)

# Train, validate, and export
model.train(data="coco8.yaml", epochs=10, imgsz=640)
metrics = model.val(data="coco8.yaml", imgsz=640)
model.export(format="onnx", imgsz=640)
```

## Task examples

```bash
# Detection
yolo predict model=yolo26n.pt source=https://ultralytics.com/images/bus.jpg

# Instance segmentation
yolo segment predict model=yolo26n-seg.pt source=https://ultralytics.com/images/bus.jpg

# Semantic segmentation
yolo semantic predict model=yolo26n-sem.pt source=https://ultralytics.com/images/bus.jpg imgsz=1024

# Depth estimation
yolo depth predict model=yolo26n-depth.pt source=https://ultralytics.com/images/bus.jpg

# Classification
yolo classify predict model=yolo26n-cls.pt source=https://ultralytics.com/images/bus.jpg

# Pose estimation
yolo pose predict model=yolo26n-pose.pt source=https://ultralytics.com/images/bus.jpg

# Oriented bounding boxes
yolo obb predict model=yolo26n-obb.pt source=https://ultralytics.com/images/boats.jpg

# Multi-object tracking
yolo track model=yolo26n.pt source=https://youtu.be/LNwODJXcvt4 tracker=botsort.yaml
```

## Useful model and dataset links

- Detection task: https://docs.ultralytics.com/tasks/detect
- Instance segmentation task: https://docs.ultralytics.com/tasks/segment
- Semantic segmentation task: https://docs.ultralytics.com/tasks/semantic
- Depth task: https://docs.ultralytics.com/tasks/depth
- Depth launch and evaluation guidance: #25695
- Classification task: https://docs.ultralytics.com/tasks/classify
- Pose task: https://docs.ultralytics.com/tasks/pose
- OBB task: https://docs.ultralytics.com/tasks/obb
- COCO8 quick test dataset: https://docs.ultralytics.com/datasets/detect/coco8
- Cityscapes semantic dataset: https://docs.ultralytics.com/datasets/semantic/cityscapes
- Model comparison pages: https://docs.ultralytics.com/compare/yolo26-vs-yolo11

## Need help?

For usage questions, include your command, model name, dataset YAML, and `yolo checks` output. For bugs, please open a focused issue with a minimum reproducible example: https://docs.ultralytics.com/help/minimum-reproducible-example

YOLO26 is the recommended starting point for new Ultralytics projects. Older YOLOv3 and YOLOv5 repositories remain available for legacy workflows, but new models and active development live here in `ultralytics/ultralytics`.

## Export status — 2026-09-05

CoreAI export is supported via #25926 on its supported macOS 26 / Python 3.11–3.13 configuration. The required CoreAI dependency remains prerelease; follow #25967 for the stable-release status. Older comments below saying CoreAI export is unsupported predate that implementation.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @glenn-jocher, thank you for opening this YOLO26 documentation tracking issue 🚀 This is an automated response to help with triage, and an Ultralytics engineer will also assist soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and where many of the most common questions may already be answered.

For this documentation-focused issue, please continue to add clear notes on any gaps around model selection, pretrained weights, training arguments, export behavior, deployment paths, and reproducibility details 📝

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example) to help us debug it. For code bugs, opening a separate focused report with an MRE will help us triage it efficiently.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by ldenoue** [comment]
is YOLO26 exportable to Apple's new Core AI?

I saw their repo but unfortunately only 2 models are currently supported:
https://github.com/apple/coreai-models/tree/main/models/yolo

Laurent

**Comment by glenn-jocher** [comment]
Yes for Apple **Core ML** today: `yolo export model=yolo26n.pt format=coreml quantize=8` produces a `.mlpackage` that can run on Apple devices, and the [Ultralytics CoreML guide](https://docs.ultralytics.com/integrations/coreml) covers the recommended iOS/Flutter deployment path. Apple **Core AI** `.aimodel` export is a separate Apple toolchain, and it is not currently a built-in Ultralytics export target, so for YOLO26 on Apple hardware CoreML is the supported path right now.
